applications/polkascan-cli.md

```diff
@@ -0,0 +1,93 @@
+
+#### Language Used
+We will be using **Python** language since we will be needing support of **py-substrate-interface**.
+We will be using **Typer** to implement the CLI: https://github.com/tiangolo/typer.
+
+#### CLI Flags
+
+There are going to be following options when users open up the CLI:
+ - Get current block number.
+ - Get Blocks: Number, time, hash, Extrinsics, Events.
+ - Get block info: Timestamp UTC, Hash, Parent Hash, State root, Extrinsics root, Extrinsics count, Events count, Runtime.
+ - Get Extrinsic details.
+ - Get account info.
+
+### Ecosystem Fit
+ - Where and how does your project fit into the ecosystem?
+Anyone who is looking for account/block/event/transaction details.
+ - Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+Our target users are ones who wants to take a look at transaction details on blockchain quickly using CLI.
+  
+
+### Simillar platforms
+
+- None so far
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+* @avirathecrawler
+
+### Legal Structure
+
+*  **Registered Legal Entity:** Not yet registered
+
+### Team's experience
+
+* avirathecrawler: Did a web3 based startup in early 2016 when ethereum was just launched. Before that, worked with IBM as a software engineer.
+
+### Team Code Repos
+
+* https://github.com/avirathecrawler
+
+## Development Roadmap :nut_and_bolt:
+
+# Overview
+
+## Grant level
+
+Level 1: Up to $10,000, 2 approvals
+
+
+## Development Roadmap :nut_and_bolt:
+
+*  **Total Estimated Duration:** 2 weeks
+
+*  **Total Costs:** 4,000 USD
+
+### Milestone 1
+
+*  **Estimated Duration:** 2 weeks
+
+*  **Costs:** 4,000 USD
+
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | Apache License 2.0 | All code will be published under Apache 2.0 |
+| 0b. | Documentation | We will provide both inline documentation of the code. |
+| 0c. | Testing and it's Guide | Core functions will be fully covered by comprehensive unit tests written in Jest and UI tests in Cypress to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Articles | We will publish a series of articles on how to contribute and create new boxes. |
+| 0. | Creation CLI tool | CLI tool will be able to fetch account, block, event, transaction and logs as on explorer.polkascan.io. CLI will be coded in Python since we might be using py-substrate-interface: https://github.com/polkascan/py-substrate-interface and polkascan API. CLI will be having interactive menu where user can navigate using arrow keys. |
```

Comment:
My understanding is that polkascan would prefer not to have external tools access their backend.  Have you gotten their permission, or could you use py-substrate to query for these directly?

Reply:
We will be spinning up our instance of Polka-explorer.